mlflow/utils/autologging_utils.py

```diff
@@ -306,6 +310,36 @@ def batch_metrics_logger(run_id):
     batch_metrics_logger.flush()
 
 
+def _check_autologging_tested(module_name, lib_key, version_file_json):
+    min_version = version_file_json[lib_key]["autologging"]["minimum"]
+    max_version = version_file_json[lib_key]["autologging"]["maximum"]
+
+    pkg_ver = importlib.import_module(module_name).__version__
+
+    def get_minor_version(v):
+        return LooseVersion(v).version[:2]
+
+    return get_minor_version(min_version) <= get_minor_version(pkg_ver) \
+        <= get_minor_version(max_version)
```

Comment:
As discussed offline, can we amend this logic to include micro version discrepancies? I.e. `0.23.3` should be considered untested if `0.23.2` was the last tested library version prior to an MLflow release.

Reply:
OK.

One issue to clarify, in this case, should `0.23.1` regarded as tested or not ? @dbczumar @harupy

Reply:
we should consider `0.23.1` to be tested, since we would’ve tested it at some point in the past prior to the release of `0.23.2`